docs/source/search-runs.rst

```diff
@@ -93,139 +93,183 @@ Then paste the following code to a local pythnon file, e.g., ``~/create_testing_
+like user name, team, etc. They are logged manually via ``mlflow.set_tag``
+or ``mlflow.set_tags``. In addition, `system tags <https://mlflow.org/docs/latest/tracking/tracking-api.html#system-tags>`_,
+such as ``mlflow.user``, are automatically logged.
 
-Searching by params requires you to write queries with key ``tags.{tag_name}``, where ``tag_name`` are the tags you logged
-to MLflow through ``mlflow.set_tag``, ``mlflow.set_tags`` or the UI. In additions, system tags such as ``mlflow.user`` are
-automatically logged, you can find the full list `here <https://mlflow.org/docs/latest/tracking/tracking-api.html#system-tags>`_.
-and these tags' value cannot be customized. The value to be compared must be a string, because MLflow tags are logged in
-string format. See below for sample queries.
+To search for runs by filtering on tags, you must include the ``tags`` or ``mlflow`` prefixes in
+the left side of the inequality. Note that tags are **stored as strings**, so you must use string 
+comparators, such as ``=`` and ``!=``.
 
 .. code-block:: sql
 
-    tags.task IN ('classification', 'regression')
-    tags."environment" == "notebook"
-    # Change this to match your user name.
-    tags."mlflow.user" == "Monkey D Luffy"
+    tags."environment" = "notebook"
+    tags.environment = "notebook"
+    tags.task = "Classification"
+    tags.task ILIKE "classif%"
 
-Searching By Dataset Information
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+4 - Searching By Dataset Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Searching by params requires you to write queries with key ``datasets.{param_name}``, and there are 3 fields you can filter
-by:
+Datasets represent data used in model training or evaluation, including features, targets, 
+predictions, and metadata such as the dataset's name, digest (hash) schema, profile, and source. 
+They are logged via ``mlflow.log_input`` or automatically via autologging.
 
-1. Dataset name ``datasets.name``, which is the dataset's name.
-2. Dataset digest ``datasets.digest``, which is a unique identifier for the dataset.
-3. Dataset context ``datasets.context``, which represents if the dataset is used for train, evaluation or test.
+To search for runs by filtering on dataset information, you must filter on one of the below fields
 
-Please note that the value to compare must be string.
+1. ``datasets.name``, which is the dataset's name.
+2. ``datasets.digest``, which is a unique identifier for the dataset.
+3. ``datasets.context``, which represents if the dataset is used for train, evaluation or test.
+
+Note that dataset information is **stored as strings**, so you must use string comparators, such as ``=`` and ``!=``.
+Also note that datasets support set comparators, such as ``IN``.
 
 .. code-block:: sql
 
     datasets.name like "custom"
     datasets.digest IN ('s8ds293b', 'jks834s2')
-    datasets.context == "train"
+    datasets.context = "train"
```

Comment:
For line 255 above, can we fix the DML statement to be capitalized? While we do support "like", standard ANSI SQL lint formatting dictates that DML operators are all CAPS